docs/reference/changelog.rst

```diff
@@ -70,23 +70,123 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
-8.6.0 (2025-Jan-20)
+8.6.0 (2025-Feb-05)
 -------------------
 
 Core
 ====
 
-* Improved error messaging when unable to pack.
+.. note::
+
+  Unless stated otherwise, the changes in this section apply to snaps using
+  ``core22`` or ``core24`` bases.
+
+* Add ``chisel`` as a ``build-snap`` if any part has slices and ``chisel``
+  isn't already listed as a ``build-snap``.
 
 Plugins
 #######
 
+Ant
+"""
+
+* Set ``JAVA_HOME`` environment variable to allow the
+  :ref:`Ant plugin <craft_parts_maven_plugin>` to detect the latest available
+  JDK.
+
+Go Use
+""""""
+
+* Add a new :ref:`Go Use plugin<craft_parts_go_use_plugin>` for setting up a
+  `workspace <https://go.dev/ref/mod#workspaces>`_ for Go modules.
+
+Maven
+"""""
+
+* Set ``JAVA_HOME`` environment variable to allow the
+  :ref:`Maven plugin <craft_parts_maven_plugin>` to detect the latest available
+  JDK.
+
+Rust
+""""
+
+* Fix dependency validation for the ``rust`` plugin when a ``rust-deps``
+  part exists (`canonical/craft-parts#890`_).
+
+Poetry
+""""""
+
+* Add new ``poetry-export-extra-args`` and ``poetry-pip-extra-args`` keys
+  (`canonical/craft-parts#932`_).
+
+* Fix an issue where the :ref:`poetry plugin <craft_parts_poetry_plugin>` would
+  try to install poetry from the package repositories when ``poetry-deps`` was
+  declared as a dependency (`canonical/charmcraft#1996`_).
+
 uv
 ""
 
-* Add a new plugin for projects using `uv <https://docs.astral.sh/uv/>`_.
-  For more information, see :doc:`uv plugin reference
-  </reference/plugins/uv_plugin>`.
+* Add a new :ref:`uv plugin<craft_parts_uv_plugin>` for projects using
+  `uv <https://docs.astral.sh/uv/>`_.
+
+Extensions
+##########
+
+KDE Neon 6
+""""""""""
+
+* Add support for ``core24`` snaps.
+
+* Fix a bug where themes defined in ``$SNAP_REAL_HOME/.config/kdeglobals``
+  would not be loaded.
+
+ROS 2 Jazzy
+"""""""""""
+
+* Remove the experimental flag from the ROS 2 Jazzy extension.
+
+Command line
+============
+
+* Show error details in all verbosity modes except ``quiet``.
```

Comment:
Since this is a long-form argument and we're quoting it, what are your thoughts on formatting it as `--quiet`?

Reply:
I agree in principle, but `quiet` can be defined with `-q`, `--quiet`, `--verbosity=quiet`, or `CRAFT_VERBOSITY=quiet`, so `quiet` focuses on the level itself rather than the way it is defined.